tools/RAiDER/processWM.py

```diff
@@ -41,34 +40,27 @@ def prepareWeatherModel(
     """
 
     # Ensure the file output location exists
-    if wmLoc is None:
-        wmLoc = os.path.join(os.getcwd(), 'weather_files')
-    os.makedirs(wmLoc, exist_ok=True)
+    ll_bounds = weather_model.get_latlon_bounds()
+    wmLoc     = weather_model.get_wmLoc()
+    weather_model.setTime(time)
+
+    # get the path to the less processed weather model file
+    path_wm_raw = make_raw_weather_data_filename(wmLoc, weather_model.Model(), time)
 
-    # check whether weather model files are supplied or should be downloaded
-    f = weather_model.filename(time, wmLoc)
+    # get the path to the more processed (cropped) weather model file
+    path_wm_crop = weather_model.out_file(wmLoc)
```

Comment:
use the cropped not raw file for checking if it exists. an existing uncropped weather model file may or may not cover the bounding box desired